pkgs/top-level/all-packages.nix

```diff
@@ -20507,7 +20512,12 @@ with pkgs;
     withLapack = false;
   };
 
-  blas = callPackage ../development/libraries/science/math/blas { };
+  # We use linkName to pass a different name to --with-blas-libs for
+  # fflas-ffpack and linbox, because we use blas on darwin but openblas
+  # elsewhere.
+  # See see https://github.com/NixOS/nixpkgs/pull/45013.
+  blas = (callPackage ../development/libraries/science/math/blas { })
+    .overrideAttrs(oldAttrs: { passthru.linkName = "blas"; });
```

Comment:
I think I didn't explain theis properly, there's no need for overrides in all-packages.nix. Add the overrides eg. here, similar to meta this won't cause a rebuild when changed. https://github.com/NixOS/nixpkgs/blob/55c4572dd1fae7043b1697116007d0f35edbcb64/pkgs/development/libraries/science/math/blas/default.nix#L55

Reply:
So you can directly add this to `pkgs/development/libraries/sciene/math/{open,}blas/default.nix`.

Reply:
Yeah sorry, makes sense now.